tinkerbell/smee/templates/deployment.yaml

```diff
@@ -87,7 +86,6 @@ spec:
             - -dhcp-syslog-ip={{ .Values.dhcp.syslogIp }}
             - -dhcp-tftp-ip={{ .Values.dhcp.tftpIp }}
             - -dhcp-ip-for-packet={{ .Values.dhcp.ipForPacket }}
```

Comment:
I've traced breakage in 0.6+ to this in hostNetwork auto-detect IP mode. It seems it causes `""` empty string passed if ipForPacket (which is defaulted to publicIP) is empty; this in turn is not caught in Smee flag parsing code leading to 
```
panic: failed to create dhcp listener: invalid bind address: ParseAddr(""): unable to parse IP
```

Reply:
Hey @rpardini , ah yeah, apologies for this! I think this PR should help resolve this. https://github.com/tinkerbell/charts/pull/150